api/services/auth.js

```diff
@@ -136,28 +141,30 @@ async function registration({ firstName, lastName, email, password, token }) {
 
 const resetPasswordRequest = async ({ email }) => {
   try {
-    let v = await db.Users.findOne({ where: { email } });
-    let linkId = v && (await hashHelpers.createHash(email));
-
-    let forgotPassword =
-      v &&
-      (await db.ForgotPassword.findOrCreate({
-        where: { linkId },
-        defaults: { UserId: v.id },
-      }));
-    let linkId2 = encodeURIComponent(linkId);
-    const info =
-      v && linkId && forgotPassword
-        ? await emailHelpers.sendEmail(
-            email,
-            `Follow link http://localhost:3000/reset?id=${linkId2}`,
-            `http://localhost:3000/reset?id=${linkId2}`
-          )
-        : { message: 'User not founded', status: 400 };
-    console.log(nodemailer.getTestMessageUrl(info));
-    return info;
+    let user = await db.Users.findOne({ where: { email } });
+    if (user) {
+      let linkId = (await hashHelpers.createHash(email));
+      let forgotPassword =
+        (await db.ForgotPassword.findOrCreate({
+          where: { linkId },
+          defaults: { UserId: user.id },
+        }));
+      let linkId2 = encodeURIComponent(linkId);
```

Comment:
Use sensible variables, what is `linkId2`?

Reply:
it ls devilry. I'll fix it